src/views/components/NameInput/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Input } from './styled';
+
+export const NameInput: React.FC<{ placeholder?: string }> = ({
```

Comment:
better to name it Input and make it generic input component. It means type placeholder etc should not be defined here. As I see you only applied styles and there is no logic so you can not to create react component and just create styled Input component